pysat/_instrument.py

```diff
@@ -2017,12 +2017,29 @@ def _filter_netcdf4_metadata(self, mdata_dict, coltype, remove=False):
         if (coltype == type(' ')) or (coltype == type(u' ')):
             # if isinstance(coltype, str):
             remove = True
+            warnings.warn('FillValue is not an acceptable '
```

Comment:
`warn` has been deprecated, use `warning`

Reply:
Here and elsewhere.

Reply:
Thanks for the note. Updated.